eole/utils/scoring_utils.py

```diff
@@ -102,6 +137,11 @@ def translate(self, model, gpu_rank, step):
         # ########### #
         # Predictions #
         # ########### #
+        if not is_seq2seq:
+            translator.id_tokenization = True
```

Comment:
well not sure to understand, what about when we don't use HF tokenizers ?

Reply:
It is supposed to work as well with others tokenizers, no ?